installers/agent/release/agent.sh

```diff
@@ -63,14 +63,36 @@ location of your Java installation."
   echo "$java_cmd"
 }
 
+declare -a _stringToArgs
+function stringToArgsArray() {
+  _stringToArgs=("$@")
+}
+
+function autoDetectGoServerUrl() {
+  local url
+
+  if [[ ! -z "${GO_SERVER}" || ! -z "${GO_SERVER_PORT}" ]]; then
+    yell "The environment variable GO_SERVER and GO_SERVER_PORT has been deprecated in favor of GO_SERVER_URL. Please set GO_SERVER_URL instead to a https url (https://example.com:8154/go)"
```

Comment:
Could you mark it as a `WARN` since the agent continues to run even after this message. Also, the message is echoed to console which could be missed if there is no user interaction while starting the service. Could we echo this to the stdout file too.